core/rx-java-driver/src/main/java/org/hawkular/rx/cassandra/driver/RxSessionImpl.java

```diff
@@ -38,6 +42,7 @@
 public class RxSessionImpl implements RxSession {
 
     private Session session;
+    private static int MAXIMUM_INFLIGHT_REQUESTS = 1024;
```

Comment:
Currently this is what we're using with the driver (after more than 1024, we start pushing to the buffer pool of the driver). 

And I couldn't find a way to request the current setting from anywhere (and DataAccessImpl can't see anything else than Session)